auth/README.md

```diff
@@ -0,0 +1,69 @@
+# Auth Package
+
+Auth package is used for request authentication and authorization for user to service
+and service to service communication. Auth is used for scenarios that require stricter
+validations, and usage restrictions. `auth.Client` provided by the package provides seamless
```

Comment:
Alternatives (to avoid provided…provides):
- `auth.Client` comes with the package to provide
- `auth.Client`, included in the package, provides